signalr-node-server/app.js

```diff
@@ -1,31 +1,41 @@
 const express = require('express');
+const cors = require('cors');
 const app = express();
+const settings = require('./settings.json');
+const corsOptions = {
```

Comment:
Undo this, I'm not sure I want to bake this into the library.

Reply:
Aight, I'll remove it